src/ConfigResource.php

```diff
@@ -329,10 +382,12 @@ protected function deleteByKey(&$array, array $keys)
             return;
         }
 
-        if (! $keys) {
+        if (1 > count($keys)) {
```

Comment:
... and this?

Reply:
I don't write code like that.  I think the reordering of protected functions to the end of the class moved these two together in the review.

Reply:
This is why we shouldn't do improvements with bug fixing because it's really confusing especially when you look on diff ...